CMakeLists.txt

```diff
@@ -25,8 +25,17 @@ add_subdirectory(embedded/peripherals)
 add_subdirectory(embedded/utils)
 add_subdirectory(middleware)
 
+
 # Do not compile tests when cross-compiling until we determine best way to include cross compiled GTest libs
 # Cross-compiled system name is "BeagleBone"
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     add_subdirectory(Test)
+    set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMakeFiles/modules)
+if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Not sure if I'm missing something, but can't these be in the same conditional? Do we need separate ones?

Reply:
The guide I'm following has two different ones. I'd like to keep it there in case a problem comes up down the road that stems from this.